### PR TITLE
Preparations for initial release

### LINKED ISSUE #6: implement xst get
```
xst get <source> [<target>]
```
followup of https://github.com/eXist-db/node-exist/issues/209

**Use Cases**

- Download an entire collection from exist to current working directory
  `xst get /db/temp`
- Download an entire collection from exist to `target` directory
  `xst get /db/temp /tmp`
- Download specific resource to target folder with the same name (defaults to current working directory if not set)
  `xst get /db/temp/resource.xml`
  `xst get /db/temp/resource.xml ~/test`
  `xst get /db/temp/resource.xml ./subfolder`
- Download specific resource to local file
  `xst get /db/temp/resource.xml test.xml`
  `xst get /db/temp/resource.xml ./test.xml`
  `xst get /db/temp/resource.xml ~/test.xml`

Possible options

- `include` - globbing pattern, only files matching the pattern are downloaded
- `exclude` - globbing pattern, files matching the pattern are excluded
- allow to set serialisation options for XML and JSON resources
- parallel downloads with X-threads

### LINKED ISSUE #9: create command to remove collections and resources
Remove files in an existdb instance, mimicking the unix `rm` program.

```
xst rm <resource> [options]
```

Options

- `--force`, `-f`: force removal of collections that are not empty
- `--recursive`, `-r`: traverse the entire collection tree when removing
- `--glob`, `-g`: only remove items that match the given glob pattern
- `--dry-run`, `-d`: show which items would be removed

refs https://github.com/eXist-db/node-exist/issues/208

### LINKED ISSUE #11: xst install removes XAR packages after installation
refs https://github.com/eXist-db/node-exist/issues/241

### LINKED ISSUE #33: create command to update installed packages
**Is your feature request related to a problem? Please describe.**

As a database admin I want to quickly update all installed packages.

**Describe the solution you'd like**

`xst package update [--repo] [<packages>..]`

`packages` is a space separated list of identifiers. Identifiers can be either the (abbreviated) name or the namespace URI of the package.

update all

- `xst package update`

update one

- `xst package update monex`
- `xst package update http://line-o/xq/dicey`

use non-default repository for lookups

- `xst package update --repo https://localhost:8080/exist/apps/public-repo monex`

**Describe alternatives you've considered**

To avoid having to move `xst install` to `xst package install` the command could also just be `xst update`.

**Additional context**

This feature request is a result of a [slack discussion](https://exist-db.slack.com/archives/CG2MRUZ35/p1662555892413129)
[A first implementation for the db query](https://gist.github.com/line-o/8bab558790a532939432da5cd3ad8ef7)

### LINKED ISSUE #34: change `xst install` to `xst package install`
related #33

### PR DESCRIPTION
**Pull request by line-o**
This will set the package name to `@existdb/xst`.
Merging this PR will trigger the release to npm which currently fails due to a naming conflict.

closes #6, closes #9, fixes #11, closes #18, closes #19, closes #20, closes #23, closes #24, closes #29, closes #33, closes #34, closes #35, closes #36, closes #38, closes #42, closes #43, closes #44, closes #45 

should be closed first

- [x] #24 
- [x] #23 
- [x] #6
- [x] configure opencollective
- [x] move `install` to `package install`
- [x] documentation

### CODE REVIEW

**Comment by duncdrum** [comment]
@line-o yup looks good

File: readme.md [hunk 1]
```diff
@@ -166,3 +213,40 @@ That also works when running the tests (on a remote server maybe or a different
...
+
+## Testing
+
+Once you followed the above steps and linked a local clone of this repository 
+
+```bash
+npm test
+```
+
+willrun the testsuite.
```
Comment (joewiz) [inline readme.md:250]: s/willrun/will run

File: readme.md [hunk 2]
```diff
@@ -166,3 +213,40 @@ That also works when running the tests (on a remote server maybe or a different
...
+    ```bash
+    npm install
+    ```
+
+4. link this package into your global packages directory
+    ```bash
+    npm link
+    ```
+
+**Note:** With this ssetup `--version` will always output `0.0.0-development`.
```
Comment (joewiz) [inline readme.md:240]: s/ssetup/setup

**Comment by joewiz** [comment]
p.s. Any idea why xst returns the following results?

```bash
% xst run '(1 to 10) ! string-join(., "&#10;")' 
1,2,3,4,5,6,7,8,9,10
```

I'd have expected the numbers to be separated by newlines. I assume this is by design but can't explain why/how this is, so I thought I'd raise it here in case it's unexpected to you too.

**Comment by line-o** [comment]
Your first query should work too.

**Comment by line-o** [comment]
@joewiz Ah, no I see why the first one could not work. :)

**Pull request merged**